languages/README.md

```diff
@@ -0,0 +1,165 @@
+of that language.
+
+Reviewers should track translation state of each document by editing 
+`languages/{language-code}/translations.md` file.
+
+There are 4 types of status:
+- Ready to start: any document is ready to be translated stay in this state.
+- Started: indicating someone is translating this document
+- PR reviewing: a translator finished the translation, and the work is being reviewed.
+- PR merged: the translation is successfully merged into the spec.
+
+Initially, the last edit time will be the time when the translation is merged.
+Reviewers should ensure translations are up-to-date with the English version as it changes, meanwhile 
+update the last edit time as well. 
+
+The following rules govern adding and removing reviewers:
+
+- Reviewers can be proposed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+- Reviewers can be removed via a PR to edit the `languages/{language-code}/reviewer.md` file.
+
+## How to add a new language 
+
+This section is mainly for anyone who needs to build a new language structure.
+
+Fork [cloudevents/spec](https://github.com/cloudevents/spec/tree/main)
+to your Github repo. 
+
+Then check the [language list](languages.md) to see if the language
+you'd like to translate into already exists. If it doesn't exist, you can start to add a new one.
+
+### Step 1. build the structure
+
+Create a directory under `languages` directory, named after the specific 
+[language code](http://www.lingoes.net/en/translator/langcode.htm).
+
+Add a new item to the [language list](languages.md), which linking to the directory 
+you just created. 
+
+### Step 2. copy documents to right location
+
+Copy all original documents except `languages` directory to `languages/{language-code}`.
+There is no need to copy project-related files(i.e.,LICENSE, OWNERS).
+
+For example:
+```
+copy cloudevents/spec.md to languages/zh-CN/cloudevents/spec.md
+copy README.md to languages/zh-CN/README.md
+```
+
+Clear content of copied documents to indicate that the document is ready to be translated.
+
+Then copy the [reviewer list](zh-CN/reviewers.md) and the [translation list](zh-CN/translations.md)
+to `languages/{language-code}`.
+
+Check example structure [here](zh-CN).
+
+## How to publish translation tasks
+
+### Step 1. open a single issue referring to an untranslated document
+
+Create an issue with label "{language-code}-translation" to publish a translation assignment.
+
+Examples:
+```
+Issue tile:        docs: translate cloudevents/spec.md to zh-CN
+Issue content:   
+                   original document: 
+                   https://github.com/cloudevents/spec/blob/main/README.md
+                   target document: 
+                   https://github.com/cloudevents/spec/blob/main/languages/zh-CN/README.md
+```
+
+A reviewer can assign the issue to anyone who comments to do the translation.
+
+### Step 2. open an overall issue to list and track all tasks
```

Comment:
s/open/Open/

Reply:
is this a different issue from step 1?

Reply:
Yes they're different. We should open an issue referring to every single untranslated document. In other words, we open 10 single issues if there are 10 untranslated docs. Every step 1 issue contains information about `original document` and `target document`.  Generally speaking it'll look like this:
```
Issue tile:        docs: translate cloudevents/spec.md to zh-CN
Issue content:   
                   original document: 
                   https://github.com/cloudevents/spec/blob/main/README.md
                   target document: 
                   https://github.com/cloudevents/spec/blob/main/languages/zh-CN/README.md

```
As for step 2 issue, there should be only 1 that kind of overall issue. It provides an overall view to all step 1 issues.
So, it might look like this:
```
Issue tile:         docs:  translation issues list
Issue content:   
                    cloudevents: 
                    Issue 1 url
                    Issue 2 url
                    Issue 3 url
     
                    cesql: 
                    Issue 4 url
                    Issue 5 url
                    Issue 6 url

                    ......
```